packages/micropip/src/micropip/_micropip.py

```diff
@@ -365,7 +397,25 @@ def install(requirements: Union[str, List[str]], keep_going: bool = False):
     )
 
 
-__all__ = ["install"]
+def _list():
+    """Get the dictionary of installed packages through micropip.
+
+    Returns
+    -------
+    ``PackageDict``
```

Comment:
Maybe to link to the correct section int the docs
```suggestion
    packages : {class}`~micropip.package.PackageDict``
```